build/windows/package_msi.ps1

```diff
@@ -0,0 +1,64 @@
+<#
+    .SYNOPSIS
+        This script creates the win .MSI
+#>
+param (
+    # Target architecture: amd64 (default) or 386
+    [string]$integration="none",
+    [ValidateSet("amd64", "386")]
+    [string]$arch="amd64",
+    [string]$tag="v0.0.0",
+    [string]$pfx_certificate_base64="none",

```

Comment:
`pfx_certificate_base64` isn't used anywhere. Was the goal to use it to create `wincert.pfx`?

This can be done:
```
[IO.File]::WriteAllBytes("wincert.pfx", [Convert]::FromBase64String($pfx_certificate_base64))
```